data/location_gen7.json

```diff
@@ -102,9 +102,32 @@
   "184": "Exeggutor Island",
   "186": "Battle Tree",
   "188": "Aether Paradise",
-  "190": "Ultra Space",
+  "190": "Ultra Space / Ultra Deep Sea",
```

Comment:
Should we just say "Ultra Deep Sea" here? It's a bit more precise, and it's unlikely to cause confusion because nothing is catchable there in SM anyway.

Reply:
Same thing for "110": "Secluded Shore / Ula'ula Beach".

Reply:
With that one it might be a bit different because stuff is actually catchable at Secluded Shore due to the berry tree.

Reply:
Having both names there would be odd, though.

I guess we'd either have to pick one, or have it check for origin game to figure out which to display.

Reply:
On second thought, both would be valid hatch locations anyway.

I think we should use the USUM version in both cases, since that's how it would display in USUM regardless of the origin game.

Reply:
If we're going with just one, I'd prefer the USUM version. Otherwise, we'd have to implement a way to request which game's info should be displayed (which I don't mind if someone does, but it probably won't be me :p)